src/app/modules/time-entries/pages/time-entries.component.ts

```diff
@@ -216,4 +235,15 @@ export class TimeEntriesComponent implements OnInit, OnDestroy {
       });
     }
   }
+
+  private rerenderDataTable(): void {
+    if (this.dtElement && this.dtElement.dtInstance) {
+      this.dtElement.dtInstance.then((dtInstance: DataTables.Api) => {
+          dtInstance.destroy();
+          this.dtTrigger.next();
```

Comment:
@edgardavid2015  Is necessary coverage this line with an unit test or update the current test

Reply:
I already created the unit test for this lines, thank you so much for the advice and the revision.